OrbitCore/Pdb.cpp

```diff
@@ -18,11 +19,16 @@ Pdb::Pdb(uint64_t module_address, uint64_t load_bias, std::string file_name,
   m_Name = Path::GetFileName(m_FileName);
 }
 
+//-----------------------------------------------------------------------------
+void Pdb::SetModulePathAndAddress(Function* func) {
+  func->set_loaded_module_path(GetLoadedModuleName());
+  func->set_module_base_address(GetHModule());
+}
+
 //-----------------------------------------------------------------------------
 void Pdb::AddFunction(const std::shared_ptr<Function>& function) {
   functions_.push_back(function);
-  functions_.back()->SetModulePathAndAddress(GetLoadedModuleName(),
-                                             GetHModule());
+  SetModulePathAndAddress(functions_.back().get());
```

Comment:
Hey, this looks all very good to me, would you just do me one favor while you are at all of this. 
This `SetModulePathAndAddress` should not be a weird side effect of `Pdb::AddFunction`. 

I think the only time `AddFunction` is used it in `OrbitModule::LoadSymbols` right after OrbitFunction is created via `make_shared`. 
Could you maybe add the two fields to OrbitFunction constructor (`module_base_address` and `loaded_module_path`) and get rid of `SetModulePathAndAddress` completely?

Reply:
I just realized its called again in `Pdb::ProcessData` which is probably unnecessary

Reply:
Yes, OrbitFunction is created only in OrbitModule and it seems like we know both fields when we call the constructor. I added commit with these changes. Thanks a lot!